packages/editor/README.md

```diff
@@ -483,6 +483,10 @@ The module also checks for sessionStorage support and conditionally exports the
 
 > **Deprecated** since 5.3, use `wp.blockEditor.MediaPlaceholder` instead.
 
+### mediaSideload
```

Comment:
not sure I understand why this is exported from the package.

Reply:
Right, there is no need.